lib/wab/impl/model.rb

```diff
@@ -174,12 +183,30 @@ def format_obj(format, ref, rid, obj)
         end
       end
 
+      def load_files()
+        unless @dir.nil?
+          Dir.foreach(@dir) { |fn|
+            next if '.' == fn[0]
+            ref = fn[0..-6]
+            path = File.join(@dir, fn);
+            @map[ref.to_i(16)] = Data.new(Oj.load_file(path, mode: :wab), true)
+          }
```

Comment:
instead of assigning to variable `path` for each entry, prefer passing it directly.. also semi-colons not required :slightly_smiling_face: 

```diff
- path = File.join(@dir, fn)
+ @map[ref.to_i(16)] = Data.new(
+   Oj.load_file(File.join(@dir, fn), mode: :wab), true
+ )
```